docs/build/_index.md

```diff
@@ -1,9 +1,164 @@
 ---
 title: "Build"
 linkTitle: "Build"
+description: "Configure and program your smart machine with Viam."
+image: "/services/icons/sdk.svg"
+imageAlt: "Tools and building blocks."
 weight: 410
 type: "docs"
-layout: "empty"
-empty_node: true
+no_list: true
 menuindent: true
 ---
+
+Viam uses a composable system of building blocks called {{< glossary_tooltip term_id="resource" text="resources" >}} (components and services) so you can add the hardware drivers and the high-level software functionality you need to your smart machine:
```

Comment:
Maybe this? I've removed the `(components and services)` because that's covered in the glossary tooltip. I think I don't like the "so you can add" - I think it's built that way to be composable in many different ways but not so the reader can add things. So that's what I'm trying to get towards:
```suggestion
Viam uses a composable system of building blocks called {{< glossary_tooltip term_id="resource" text="resources" >}} which you can combine according to your individual needs.
Components make your machine components usable with a consistent API and services provide high-level software funcitonality.
```

Reply:
```suggestion
Viam uses a composable system of building blocks called {{< glossary_tooltip term_id="resource" text="resources" >}} which you can combine according to your specific needs.
Components drive your hardware and services provide high-level software functionality.
```
How about this? Don't want to say "components" twice in a row.

Reply:
SGTM